content/learn.md

```diff
@@ -24,6 +24,8 @@ We highly recommend pairing R4DS with the [RStudio cheatsheets](https://www.rstu
 * [Solutions and notes for R4DS](https://jrnold.github.io/r4ds-exercise-solutions/) 
   by Jeffrey B. Arnold. Work in progress.
 
+* [Data Manipulation in R](http://geni.us/datamanipulationir) by Steph Locke. Covers data manipulation in a tidyverse way.
```

Comment:
Should that be `datamanipulationinr`?

Reply:
Made the link work :) http://geni.us/datamanipulationir